p8e-api-webservice/src/main/kotlin/io/provenance/p8e/webservice/domain/ApiAffiliate.kt

```diff
@@ -43,9 +43,10 @@ fun AffiliateShareRecord.toApi(): ApiAffiliateShare =
         created
     )
 
-data class RegisterAffiliateKey(val signingPrivateKey: String?, val encryptionPrivateKey: String?, val useSigningKeyForEncryption: Boolean, val indexName: String, val alias: String?) {
+data class RegisterAffiliateKey(val signingPrivateKey: String?, val encryptionPrivateKey: String?, val authKeyPair: String?, val useSigningKeyForEncryption: Boolean, val indexName: String, val alias: String?) {
```

Comment:
Looks like this is actually the private key in this request, right? (i.e. the name authKeyPair might be misleading)